common/src/main/java/me/shedaniel/architectury/event/events/GuiEvent.java

```diff
@@ -32,62 +32,132 @@
 
 import java.util.List;
 
+/**
+ * Should be moved to the client package and renamed to ScreenEvent in version 2.0
+ */
 @Environment(EnvType.CLIENT)
 public interface GuiEvent {
     /**
-     * Invoked after in-game hud is rendered, equivalent to forge's {@code RenderGameOverlayEvent.Post@ElementType#ALL} and fabric's {@code HudRenderCallback}.
+     * @see RenderHud#renderHud(PoseStack, float)
      */
     Event<RenderHud> RENDER_HUD = EventFactory.createLoop();
+    /**
+     * @see DebugText#gatherText(List)
+     */
     Event<DebugText> DEBUG_TEXT_LEFT = EventFactory.createLoop();
     Event<DebugText> DEBUG_TEXT_RIGHT = EventFactory.createLoop();
     /**
-     * Invoked during Screen#init after previous widgets are cleared, equivalent to forge's {@code GuiScreenEvent.InitGuiEvent.Pre}.
+     * @see ScreenInitPre#init(Screen, List, List)
      */
     Event<ScreenInitPre> INIT_PRE = EventFactory.createInteractionResult();
     /**
-     * Invoked after Screen#init, equivalent to forge's {@code GuiScreenEvent.InitGuiEvent.Post}.
+     * @see ScreenInitPost#init(Screen, List, List)
      */
     Event<ScreenInitPost> INIT_POST = EventFactory.createLoop();
+    /**
+     * @see ScreenRenderPre#render(Screen, PoseStack, int, int, float)
+     */
     Event<ScreenRenderPre> RENDER_PRE = EventFactory.createInteractionResult();
+    /**
+     * @see ScreenRenderPost#render(Screen, PoseStack, int, int, float)
+     */
     Event<ScreenRenderPost> RENDER_POST = EventFactory.createLoop();
-    
     /**
-     * Invoked during Minecraft#setScreen, equivalent to forge's {@code GuiOpenEvent}.
+     * @see SetScreen#modifyScreen(Screen)
      */
     Event<SetScreen> SET_SCREEN = EventFactory.createInteractionResultHolder();
     
     @Environment(EnvType.CLIENT)
     interface RenderHud {
+        /**
+         * Invoked after the in-game hud has been rendered.
+         * Equal to the Forge {@code RenderGameOverlayEvent.Post@ElementType#ALL} event and fabric's {@code HudRenderCallback}.
+         * 
+         * @param matrices The render buffer.
+         * @param tickDelta The delta tick.
+         */
         void renderHud(PoseStack matrices, float tickDelta);
     }
     
     @Environment(EnvType.CLIENT)
     interface DebugText {
+        /**
+         * Invoked when the debug text is being gathered for rendering.
+         * There are two different versions of this event, one for the left and one for the right side.
+         * Equal to the Forge {@code RenderGameOverlayEvent.Text} event, when {@code Minecraft.getInstance().options.renderDebug} is true.
+         * 
+         * @param strings Contains the already added strings. You can add your own lines by adding to the list.
```

Comment:
We don't need the excessive help in the parameters.

```suggestion
         * @param strings The current debug text strings.
```